lib/ember-test-helpers/isolated-container.js

```diff
@@ -83,5 +84,8 @@ export default function isolatedContainer(fullNames) {
     var normalizedFullName = resolver.normalize(fullName);
     container.register(fullName, resolver.resolve(normalizedFullName));
   }
-  return container;
+  return {
+    container: container,
+    registry: registry
+  };
 }
```

Comment:
I am wary of changing the return value of `isolatedContainer` here, especially given the likelihood of the container/registry reform work proceeding very soon (https://github.com/emberjs/rfcs/pull/46). In other words, I don't want developers who are using `isolatedContainer` to need to refactor twice in a short time. Maybe the question should be whether `isolatedContainer` is really used outside of ember-mocha and ember-qunit? 

@ef4 @rwjblue thoughts?